.editorconfig

```diff
@@ -0,0 +1,15 @@
+# Keep this file in sync with .astylerc
+
+root = true
+
+[*]
+end_of_line = "lf"
+charset = "utf-8"
```

Comment:
neovim won't work if values are quoted.

I haven't checked the actual spec, but their website shows unquoted values as well.

https://github.com/axoflow/axosyslog/blob/main/.editorconfig

Reply:
Oops, thanks for catching that!